### DISCUSSION TITLE
pH7Builder v17.0.1 is released!

### DISCUSSION THREAD

**Discussion by pH-7** [pH7Software/pH7-Social-Dating-CMS#972] [Release Announcements]
"pH7Builder v17.0.1" has just been released. In [comparison with v17.0.0](https://github.com/pH7Software/pH7-Social-Dating-CMS/releases/tag/17.0.0), this version **is a really minor one** and useful only if you install pH7Builder with `composer`.

✅ "unit test" file was always loaded when pH7Builder was installed with composer dev mode (in `autoload-dev`).
✅ Install `_install`'s dependencies when running the main composer.json https://github.com/pH7Software/pH7-Social-Dating-CMS/commit/6341e171563de5f961d045a84f314d3b7cc7c7c7
✅ Disabled by default `chatroulette` module.
✅ Exclude a few other dev files from the distribution archive.

👉 Changelog: https://github.com/pH7Software/pH7-Social-Dating-CMS/compare/17.0.0...17.0.1

Enjoy your ew

<hr /><em>This discussion was created from the release <a href='https://github.com/pH7Software/pH7-Social-Dating-CMS/releases/tag/17.0.1'>pH7Builder v17.0.1 is released!</a>.</em>

**Comment by Mistaxun** [comment]
Hi. Billing section doesn't work in admin panel
